math-scala/src/main/scala/org/apache/mahout/math/algorithms/regression/tests/FittnessTests.scala

```diff
@@ -41,16 +41,22 @@ object FittnessTests {
     val r2 = 1 - (sumSquareResiduals / sumSquareTotal)
     model.r2 = r2
     model.testResults += ('r2 -> r2)  // need setResult and setSummary method incase you change in future, also to initialize map if non exists or update value if it does
-    model.summary += s"\nR^2: ${r2}"
+    // this is already being done in  -> calculateCommonStatistics
```

Comment:
so it is- good catch.  That said, I think it _should_ happen here, and be taken out of calculateCommonStatistics (adding to the summary string)